aspnet/identity/overview/getting-started/aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider.md

```diff
@@ -60,15 +60,15 @@ To complete the steps in this section of the tutorial, you will first need to in
 2. Click **New Project** from the **Start** page, or you can click the **File** menu and then **New Project**:
 
    [Click the following image to expand it. ]
-    [![Illustrates Visual Studio Start page, with New Project selected at left.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.jpg)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.jpg)
+    [![Screenshot of Visual Studio Start page, with New Project selected in menu at left.](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image2.jpg)](aspnet-identity-using-mysql-storage-with-an-entityframework-mysql-provider/_static/image1.jpg)
```

Comment:
Too short for recommended length; consider: "Screenshot of the Visual Studio start page showing the New Project option highlighted with a red rectangle."